### PR TITLE
[WIP] Groups page redesign

### PR DESCRIPTION
**Pull request by Frenje123**

### CODE REVIEW

**Comment by Frenje123** [comment]
We would also need to create abbreviations for these groups after deploying:
* Sikringskomiteen
* S. Møller Storband
* Salongorkesteret
* Big Mama's Cowboys

File: app/views/groups/index.html.haml [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+  %h1= t("pages.index.groups")
+  %p= t("pages.index.groups_info")
 
-%h1= t("pages.index.groups")
+.groups-showcase-container
+  - @group_types.each do |t|
+    .samf-container.plain.p-3.m-2.mt-3
+      .flex-row.wrap.center
+        .group-type-headline=t
+        - t.groups.each do |g|
```
Comment (evgiz) [inline app/views/groups/index.html.haml]: If number of groups with type is empty, do not render container.
Must count number of groups without pages

**Review by evgiz — COMMENTED** [review]
ISFiT logo is stretched

**Comment by evgiz** [comment]
> We would also need to create abbreviations for these groups after deploying:
> 
> * Sikringskomiteen
> * S. Møller Storband
> * Salongorkesteret
> * Big Mama's Cowboys

Not necessary after changes

File: config/locales/views/groups/en.yml [hunk 2]
```diff
@@ -5,6 +5,7 @@ en:
     email: E-mail
     name: Name
     abbreviation: Abbreviation
+    no_abbreviation: No known abbreviation in existance.
```
Comment (evgiz) [inline config/locales/views/groups/en.yml]: abbreviation exists

File: app/assets/stylesheets/groups/index.scss [hunk 3]
```diff
@@ -0,0 +1,123 @@
+span img { display: inline; }
```
Comment (evgiz) [inline app/assets/stylesheets/groups/index.scss]: remove

**Review by evgiz — CHANGES_REQUESTED** [review]

**Review by evgiz — APPROVED** [review]
Awesome<3

**Comment by oscarahalvorsen** [comment]
closes #731
closes #732

**Pull request merged**